complex/ft_comp.c

```diff
@@ -54,8 +54,6 @@ static int ft_open_cntrs(void)
 {
 	int ret;
 
-	opts.comp_method = test_info.cq_wait_obj;
-
```

Comment:
@aingerson - I removed this.  This assignment is done in a very odd location, and it was not clear why such an association should be made, or where the correct place to make it was.

This currently breaks the build, so we need at least a quick fix for that now.  Are there new test config files for these new options, or updates to the config files?